src/node_sea.cc

```diff
@@ -284,14 +419,20 @@ void Initialize(Local<Object> target,
                 Local<Value> unused,
                 Local<Context> context,
                 void* priv) {
+  SetMethod(context, target, "isSea", IsSea);
   SetMethod(context,
             target,
             "isExperimentalSeaWarningNeeded",
             IsExperimentalSeaWarningNeeded);
+  SetMethod(context, target, "getCodePath", GetCodePath);
```

Comment:
Since these are native functions, crossing the C++-JS boundary seems to be slowing things down a little. I was trying to expose these as private symbol properties on the process object in https://github.com/nodejs/node/blob/2fca7ea2be160d8c30c5bacf7d6246e4c6f2a7f0/src/node_process_object.cc#L81 but that function is only called during the snapshot creation step and it doesn't get called when `node` starts running. Is there another place where this could be done?

Reply:
I was wrong, there is no perf improvement if we convert the native function calls into properties on the process object. I guess the changes in numbers I saw that day was just a flake.